src/get/index.js

```diff
@@ -115,7 +115,9 @@ async function get(options) {
      */
     const ffmpegFilePathExists = await util.fileExists(ffmpegFilePath);
     if (ffmpegFilePathExists === false) {
-      ffmpegFilePath = await ffmpeg(options.downloadUrl, options.version, options.platform, options.arch, options.cacheDir);
+      const downloadUrl =
```

Comment:
```suggestion
     // Do not update the options.downloadUrl with the ffmpeg URL here. Doing so would lead to error when options.ffmpeg and options.nativeAddon are both enabled.
      const downloadUrl =
```

Reply:
This should prevent my future self from changing this behaviour